### PR TITLE
Add healthcheck endpoints

### LINKED ISSUE #30: Healthcheck endpoint

### PR DESCRIPTION
**Pull request by t4ccer**
Close #30

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -429,6 +433,30 @@ Response (fault):
...
 
+#### GetHealthcheck
+Request:
+```json
+{
+  "type": "jsonwsp/request",
+  "version": "1.0",
+  "servicename": "ogmios",
+  "methodname": "GetHealthcheck",
+  // optional
```
Comment (ngua) [inline README.md:444]: Since JSON doesn't have comments this renders a big strangely

File: src/Api/WebSocket/Types.hs [hunk 2]
```diff
@@ -50,6 +52,7 @@ data Method
   | GetBlock
   | StartFetchBlocks Int64 Text DatumFilter
   | CancelFetchBlocks
+  | GetHealthcheck
```
Comment (ngua) [inline src/Api/WebSocket/Types.hs:55]: I think `HealthCheck` might be a clearer name, but that's just my personal preference

**Review by ngua — APPROVED** [review]
Some minor comments

**Review by ryukzak — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (t4ccer) [reply README.md:444]: TIL we can change `json` to `jsonc` and comments work fine 
```jsonc
// comment
"foo": "bar"
```

**Pull request merged**